### PR TITLE
readds the ability to become a snail person through genetic meltdown and the chemical agent-s

### PR DESCRIPTION
**Pull request by boodaliboo**
the reason nich made these unavaiable was fixed a while ago
#### Changelog

:cl:  
rscadd: readded the ability to become snail
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/snail.dm [hunk 1]
```diff
@@ -58,5 +58,5 @@
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 
 /obj/item/storage/backpack/snail/Initialize()
-	. = ..()
+	. = .()
```
Comment (Hopekz) [inline code/modules/mob/living/carbon/human/species_types/snail.dm]:
```suggestion
	. = ..()
```
Not sure what you're doing here.
It's like saying 1 is 1 pretty sure you meant to do this instead

Also, it should compile after this

**Review by Hopekz — CHANGES_REQUESTED** [review]

**Comment by boodaliboo** [comment]
nope

**Review by Hopekz — CHANGES_REQUESTED** [review]
pt:2
You re-included the file meaning you need to modify it to our new standards.

To make it compile:
modify line 62
from ADD_TRAIT(src, TRAIT_NODROP) to ADD_TRAIT(src, TRAIT_NODROP, CURSED_ITEM_TRAIT)

Also; please test your PR since you cannot have tested it since it didn't compile.

**Comment by boodaliboo** [comment]
> pt:2
> You re-included the file meaning you need to modify it to our new standards.
> 
> To make it compile:
> modify line 62
> from ADD_TRAIT(src, TRAIT_NODROP) to ADD_TRAIT(src, TRAIT_NODROP, CURSED_ITEM_TRAIT)
> 
> Also; please test your PR since you cannot have tested it since it didn't compile.

i truthfully thought there would be no reason to test it as it was just commenting out things disabled

**Comment by Amelia0010** [comment]
>the reason nich made these unavaiable was fixed a while ago

Please don't lie. The reason I made these unavailable is because snails are bad.

**Review by Hopekz — APPROVED** [review]

**Comment by Hopekz** [comment]
Going to let this PR marinate just in-case someone has any additional opinions on if this shouldn't be merged.

**Pull request merged**